packages/web/hooks/use-estimate-tx-fees.ts

```diff
@@ -91,6 +135,7 @@ export function useEstimateTxFees({
         accountStore,
```

Comment:
`superjson.stringify(messages)`

Should we also stringify the `sendToken`, so if that changes it triggers a new query?